CHANGELOG.md

```diff
@@ -158,12 +158,17 @@ Summary:
     ```py
     def render_to_response(
         context: Optional[Union[Dict[str, Any], Context]] = None,
-        args: Optional[Tuple[Any, ...]] = None,
-        kwargs: Optional[Mapping] = None,
-        slots: Optional[Mapping] = None,
+        args: Optional[Any] = None,
```

Comment:
Not related to this PR, except for the `node` kwarg at the end. Just realized that the rest of the signature doesn't reflect all the changes.